ddtrace/tracer.py

```diff
@@ -328,6 +336,19 @@ def wrap_decorator(f):
             else:
                 @functools.wraps(f)
                 def func_wrapper(*args, **kwargs):
+                    # if a wrap executor has been configured, it is used instead
+                    # of the default tracing function
+                    if getattr(self, '_wrap_executor', None):
+                        return self._wrap_executor(
+                            self,
+                            f, args, kwargs,
+                            span_name,
+                            service=service,
+                            resource=resource,
+                            span_type=span_type,
```

Comment:
that one should be ok: https://github.com/DataDog/dd-trace-py/pull/221/files#diff-f1664abaa28f924f71b57e9069aa70a0R225